lldb/source/Target/ThreadList.cpp

```diff
@@ -494,6 +494,49 @@ void ThreadList::DiscardThreadPlans() {
     (*pos)->DiscardThreadPlans(true);
 }
 
+void ThreadList::SetUpStepOverBreakpointBeforeResumeIfNeeded(
```

Comment:
@jimingham , I ended up removing some comments that seems not to reflect exactly the source code here. Let me know what you think

Reply:
That's better.  It still doesn't stop someone from adding so code below this that isn't breakpoint related, which will then work on every platform but yours because they didn't notice this return.  If the early return & breakpoint handling were in a separate method, then that mistake wouldn't be possible.
Not terribly important, however.